module/vision/BallDetector/src/BallDetector.cpp

```diff
@@ -73,24 +73,23 @@ namespace module::vision {
                 std::vector<int> indices(horizon.mesh->indices.size());
                 std::iota(indices.begin(), indices.end(), 0);
 
-                // Partition the indices such that we only have the ball points that dont have ball surrounding them
+                // Partition the indices such that the ball points that dont have ball surrounding them are removed,
+                // and then resize the vector to remove those points
```

Comment:
This comment has been made wrong.
We don't get the bulk of the ball points (points where the ball is surrounded by ball) we get the edges (ball points that have points around them that are not ball)

We keep the edge points and remove everything else

Reply:
Yeah lol I wrote this wrong, too many negatives for my brain apparently

Reply:
```suggestion
                // Partition the indices such that the ball points that have ball points surrounding them are removed,
                // and then resize the vector to remove those points
```